src/Controls/tests/UITests/UITest.cs

```diff
@@ -163,6 +163,13 @@ public void VerifyScreenshot(string? name = null)
 
 			name ??= TestContext.CurrentContext.Test.MethodName ?? TestContext.CurrentContext.Test.Name;
 
+			// Currently Android is the OS with the ripple animations, but Windows may also have some animations
+			// that need to finish before taking a screenshot.
+			if (_testDevice == TestDevice.Android)
+			{
+				Thread.Sleep(350);
+			}
```

Comment:
This is not amazing, but I am not yet sure the best time to wait for the ripple (I have seen 500-1000 ms), but I am sure it is less. We could drop this time to maybe 200ms, but for now I will test this number.

In addition, we might wait in the clicks/taps, but that feels like it would delay many other tests more than just waiting on the screenshots.

Not sure it is worth getting fancy to wait for the ripple to complete via some magical API. It will still take a few ms and even if we catch the cases, how much time will we save since most of the time a screenshot follows after a navigation/click anyway.